src/test/features/response/routes/confirmation.ts

```diff
@@ -96,6 +96,29 @@ describe('Defendant response: confirmation page', () => {
             'If they reject your response the court will review the case. You might have to go to a hearing.'))
       })
 
+      it('should render page when yes for mediation and DQs disabled', async () => {
+        claimStoreServiceMock.resolveRetrieveClaimByExternalIdWithResponse(claimStoreServiceMock.sampleDefendantResponseWithoutDQAndWithMediationObj)
```

Comment:
I think i already forgot the journey, will we have mediation with DQ? @robertparkinson

Reply:
I will try to answer my question, i guess mediation obj will exist with mediation as `NO` and then there will be no DQ.